### PR TITLE
Prevents Removal of Cyborgs

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Prevents the removal of cyborgs.

Don't worry, we'll all saved now.

### CODE REVIEW

**Comment by Crazylemon64** [comment]
:ok_hand:

**Comment by KasparoVy** [comment]
:ok_hand:

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -23,14 +23,12 @@
...
-	return
 
 
 /obj/item/weapon/robot_module/New()
-	src.modules += new /obj/item/device/flashlight(src)
-	src.modules += new /obj/item/device/flash/cyborg(src)
-	src.emag = new /obj/item/toy/sword(src)
-	src.emag.name = "Placeholder Emag Item"
+	modules += new /obj/item/device/flash/cyborg(src)
+	emag = new /obj/item/toy/sword(src)
```
Comment (tigercat2000) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: this is still horrifying

Comment (TheDZD) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: It really is.

**Pull request merged**